services/movie.py

```diff
@@ -28,13 +33,14 @@ def create_movie(
     genres_ids: list = None,
     actors_ids: list = None,
 ) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
+    with transaction.atomic():
```

Comment:
Consider using the `@transaction.atomic` decorator instead of `with transaction.atomic()` for the `create_movie` function. This approach is recommended for better readability and to ensure the entire function executes within a single transaction context.